hnn_core/network.py

```diff
@@ -161,13 +165,13 @@ def __init__(self, params, add_drives_from_params=False,
         # Create CellResponse object, initialised with simulation time points
         self.cell_response = CellResponse(times=times)
 
-        # Source list of names, first real ones only!
-        self.cellname_list = [
-            'L2_basket',
-            'L2_pyramidal',
-            'L5_basket',
-            'L5_pyramidal',
-        ]
+        # Source dict of names, first real ones only!
+        self.cell_types = {
+            'L2_basket': basket((0, 0, 0), cell_name=_short_name('L2_basket')),
+            'L2_pyramidal': pyramidal((0, 0, 0), _short_name('L2_pyramidal')),
+            'L5_basket': basket((0, 0, 0), cell_name=_short_name('L5_basket')),
+            'L5_pyramidal': pyramidal((0, 0, 0), _short_name('L5_pyramidal'))
+        }
```

Comment:
The user can now also load a different cell type via

```python
net.cell_types['L5_pyramidal'] = pyramdal_ca((0, 0, 0), _short_name('L5_pyramidal'))
```
where `pyramidal_ca()` is e.g. a pyramidal cell with updated calcium dynamics imported from `cells_default.py`.

Reply:
Is there ever an instance where the position wouldn't start at `(0,0,0)`? If I understand correctly, the cell at this position is copied to the other locations, so the user doesn't really have a use for that argument.

Reply:
I made `pos=(0, 0, 0,)` default in https://github.com/jonescompneurolab/hnn-core/pull/321/commits/063f99eb813b2ea784dc88ac6ab8f3ee24a3e897.